subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ResolveConfigurationDependenciesBuildOperationIntegrationTest.groovy

```diff
@@ -330,6 +330,10 @@ class ResolveConfigurationDependenciesBuildOperationIntegrationTest extends Abst
             dependencies {
                 compile 'org.foo:app-dep:1.0'
             }
+            
+            tasks.withType(JavaCompile) {
+                options.annotationProcessorPath = files()
```

Comment:
:o: Isn't the fact you had to add this a hint that this PR is potentially a breaking change?

Reply:
This is only here because this test asserts a number of resolve operations. It's not a breaking change, I just didn't care about asserting on the processor path, so I made it empty.

Reply:
The other tests were similarly adjusted to silence the deprecation warning. E.g. Antlr4 contains an annotation processor, which we don't actually want to use in that test.